CircomIntro/step-6/README.md

```diff
@@ -0,0 +1,27 @@
+With the trusted setup complete, you can now compute the witness for your circuit based on specific inputs.
+
+## What is a Witness?
+
+- A **Witness** is a set of values that satisfy all the constraints of your circuit.
```

Comment:
that requires more details I think. Like "A witness represents the execution of a program, it usually contains the actual return value of the execution + a witness that represents the execution and can be used later one to generate a zk proof that the circuit has been executed."